versions/1/changelog.yml

```diff
@@ -175,3 +175,6 @@
   - description: Introduce "value" property for constant_keyword fields
     type: enhancement
     link: https://github.com/elastic/package-spec/pull/194
+  - description: Support `ML model` Elasticsearch type.
```

Comment:
You put this in a wrong place. Please consider the order or changes in the changelog. Next version is `1.1.1-next` and this entry should be long there.